release_images.yml

```diff
@@ -2,10 +2,10 @@
 release_images:
   1:
     framework: "tensorflow"
-    version: "2.10.1"
```

Comment:
Are these changes intentional?

Reply:
As discussed offline, I synced to `master`'s `head`:
- This is the clean `diff`: https://github.com/aws/deep-learning-containers/compare/master...sheilaliuxl:deep-learning-containers:smmp.
- The extra file is likely because Shibo's `head` is not update to date yet.